### ISSUE TITLE
Email normalization

### ISSUE THREAD

**Issue by buzzlighty** [reacherhq/check-if-email-exists#952]
Add a field `misc.normalized_email` with normalized email. E.g. without gmail dots and plus signs etc.

**Comment by amaurym** [comment]
Good idea! I would even put it in `syntax.normalized_email`. @buzzlighty would you like to create a PR?

**Comment by buzzlighty** [comment]
@amaurym Yes, I may find some spare time

**Comment by PsypherPunk** [comment]
Looking at the timestamps, is there still an interest in this? Happy to raise a PR if so…?

**Comment by amaurym** [comment]
I think you can go for it. Does `syntax.normalized_email` as an `Option<String>` make sense to you?

**Comment by PsypherPunk** [comment]
What would the `None` case be? When an email is already in it's normalized form?

**Comment by PsypherPunk** [comment]
Ignore me: of course, if the syntax is invalid it should be `null` in the output like the rest.

PR incoming…
